### PR TITLE
Add log_listener and logging for pytest to support additional loggers

### PR DESCRIPTION
**Pull request by rnetser**
Required to allow multiple loggers (one with additional details, e.g date and a basic one).
This enables adding additional info to the logs - test name. setup and teardown sections

### CODE REVIEW

**Comment by rnetser** [comment]
verified

File: conftest.py [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
 BASIC_LOGGER = logging.getLogger("basic")
-LOGGER = get_logger(__name__)
+
+
+def pytest_addoption(parser):
+    log_group = parser.getgroup(name="PytestLog")
+    log_group.addoption(
+        "--pytest-log-file",
+        help="Path to pytest log file",
+        default="pytest-tests.log",
```
Comment (mwaykole) [inline conftest.py:18]: may be we should give the default path to /tmp/odh-test.log

Comment (rnetser) [reply conftest.py]: i want to keep the same convention as pytest (default location in parent dir)

**Review by adolfo-ab — APPROVED** [review]

**Pull request merged**